google-apis-core/lib/google/apis/core/http_command.rb

```diff
@@ -30,10 +30,10 @@ class HttpCommand
         RETRIABLE_ERRORS = [Google::Apis::ServerError, Google::Apis::RateLimitError, Google::Apis::TransmissionError]
 
         begin
-          require 'opencensus'
-          OPENCENSUS_AVAILABLE = true
```

Comment:
Did you want to maintain support for opencensus in addition to opentelemetry?

Reply:
Uhh, yeah, as much as I don't want to support OpenCensus here anymore, I also don't think we should rip it out unceremoniously. We should rather add a deprecation warning for a time. Can we use the logic of: try requiring opentelemetry and if that works enable it, otherwise try requiring opencensus and if that works enable it, otherwise disable both? So you can just leave the `opencensus_{begin,end}_span` methods in place, and add new ones for OpenTelemetry.